plantcv/plantcv/__init__.py

```diff
@@ -94,8 +95,8 @@
 
 __all__ = ['fatal_error', 'Params', 'Outputs', 'Spectral_data', 'deprecation_warning', 'print_image', 'plot_image',
            'color_palette', 'apply_mask', 'gaussian_blur', 'transform', 'hyperspectral', 'readimage', 'readbayer',
-           'laplace_filter', 'sobel_filter', 'scharr_filter', 'hist_equalization', 'erode', 'image_add', 
-           'image_subtract', 'dilate', 'watershed', 'rectangle_mask', 'rgb2gray_hsv', 'rgb2gray_lab', 'rgb2gray_cmyk',
+           'laplace_filter', 'sobel_filter', 'scharr_filter', 'hist_equalization', 'erode',
+           'image_add', 'image_fusion', 'image_subtract', 'dilate', 'watershed', 'rectangle_mask', 'rgb2gray_hsv', 'rgb2gray_lab',
```

Comment:
Move `rgb2gray_lab` to a new line and redistribute list to keep the line length < 120 characters